ui/selectors/confirm-transaction.js

```diff
@@ -253,6 +253,7 @@ export const transactionFeeSelector = function (state, txData) {
     }
   } else {
     switch (gasEstimateType) {
+      case GasEstimateTypes.feeMarket:
```

Comment:
Not needed in this PR, but this is a fundamental public selector with no unit tests, so worth creating a ticket to add some and maybe even refactor the selector for readability at the same time.